ceph-build-os/build/build

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+upload() {
+	IMAGE=$(ls /var/lib/debootstrap/install/$DISTRO_RELEASE-$VERSION/*qcow2)
+	if [ -z "$IMAGE" ]; then
+	   echo "upload: No IMAGE found in /var/lib/debootstrap/install/$DISTRO_RELEASE-$VERSION/, Exiting"
+	   exit 1
+	fi
+
+	IMAGE_NAME=$(basename `echo $IMAGE | sed -e "s/\.img\.qcow2//g"`)
+
+	echo "upload: About to upload $IMAGE as $IMAGE_NAME"
+
+	# If the same image already exists, let's delete it
+	openstack image list -f csv | grep -qw "$IMAGE_NAME" && echo "upload: Deleting remote image" && openstack image delete "$IMAGE_NAME"
+
+	# Upload the image
+	echo "upload: Uploading $IMAGE"
+	openstack image create $IMAGE_NAME --disk-format qcow2 --container-format bare --file $IMAGE
+}
+
+set +x
+
+export OS_AUTH_URL="$OVH_AUTH_URL"
+export OS_TENANT_ID="$OVH_TENANT_ID"
+export OS_TENANT_NAME="$OVH_TENANT_NAME"
+export OS_USERNAME="$OVH_USERNAME"
+export OS_PASSWORD="$OVH_PASSWORD"
+export OS_REGION_NAME="$OVH_REGION"
+
+if [ -z $DIST ]; then
+    echo "build: No DIST found, exiting !"
+    exit 1
+fi
+
+if [ -z $DISTRO_RELEASE ]; then
+    echo "build: No DISTRO_RELEASE found, exiting !"
+    exit 1
+fi
+
+if [ -z $VERSION ]; then
+    echo "build: No VERSION found, exiting !"
+    exit 1
+fi
+
+if [ -z "$OS_AUTH_URL" ]; then
+    echo "upload: Cannot find any valid OS_AUTH_URL"
+    exit 1
+fi
+
+if [ -z "$OS_TENANT_ID" ]; then
+    echo "build: Cannot find any valid OS_TENANT_ID"
+    exit 1
+fi
+
+if [ -z "$OS_TENANT_NAME" ]; then
+    echo "build: Cannot find any valid OS_TENANT_NAME"
+    exit 1
+fi
+
+if [ -z "$OS_USERNAME" ]; then
+    echo "build: Cannot find any valid OS_USERNAME"
+    exit 1
+fi
+
+if [ -z "$OS_PASSWORD" ]; then
+    echo "build: Cannot find any valid OS_PASSWORD"
+    exit 1
+fi
+
+if [ -z "$OS_REGION_NAME" ]; then
+    echo "build: Cannot find any valid OS_REGION_NAME"
+    exit 1
+fi
+
+echo "build: Environment is correct"
+
+if [ -z "$REPO_URL" ]; then
+    case "$DIST" in
+        "xenial"|"trusty")
+            REPO_URL="http://ubuntu.mirrors.ovh.net/ubuntu/"
+        ;;
+        "jessie")
+            REPO_URL="http://debian.mirrors.ovh.net/debian/"
+        ;;
+        "centos")
+            REPO_URL="http://centos.mirrors.ovh.net/ftp.centos.org/7/os/x86_64/Packages/centos-release-7-2.1511.el7.centos.2.10.x86_64.rpm"
+        ;;
+        *)
+            echo "build: no known favorite repository for $DIST"
+        ;;
+    esac
+    echo "build: Using repository $REPO_URL"
+fi
+
+
+sudo edeploy activate-pkgmngr
```

Comment:
same with `edeploy` 

I'm not finding where these executables are getting installed